front-end/src/components/common/MobileHeader/MobileHeader.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useContext } from 'react';
+import { Link } from 'react-router-dom';
+import Typography from '@mui/material/Typography';
+import PropTypes from 'prop-types';
+
+// Context
+import { CurrentUserContext } from 'context/AppProvider';
+
+// Assets
+import logo1 from 'assets/sidebar/logo-letter.png';
+import avatar from 'assets/sidebar/avatar.png';
+
+// Styles
+import styles from './MobileHeader.module.scss';
+import MenuIcon from '@mui/icons-material/Menu';
+
+const MobileHeader = ({ showSidebar, sidebarState }) => {
+  const { currentUser } = useContext(CurrentUserContext);
+
+  const handleClick = () => {
+    showSidebar(!sidebarState);
+  };
+
+  const userNameArr = currentUser.fullName.split(' ');
+  const cuttedUserName = `${userNameArr[0]} ${userNameArr[1][0]}.`;
```

Comment:
Is `userNameArr` matrix? What mean `userNameArr[1][0]`?

Reply:
I use userNameArr for splitting strings like this "Alex Serhieiev", and then I make this string to this "Alex S." for mobile version

Reply:
Got it. Thanks!